scholia/app/templates/author.html

```diff
@@ -500,13 +506,6 @@ <h2 id="Academic tree">Academic tree</h2>
 </div>
 
 
-<h2 id="Locations">Locations</h2>
```

Comment:
@Daniel-Mietchen, is this section intentionally removed? I cannot make up from the PR or the bug report if it is meant to replace this section.

Reply:
I have now put it back in as "other locations", after the co-author map.